content/en/docs/concepts/services-networking/_index.md

```diff
@@ -7,9 +7,7 @@ description: >
 
 ## The Kubernetes network model
 
-Every [`Pod`](/docs/concepts/workloads/pods/) in a cluster gets its own unique cluster-wide IP address. 
-This means you do not need to explicitly create links between `Pods` and you 
-almost never need to deal with mapping container ports to host ports.  
+Every [`Pod`](/docs/concepts/workloads/pods/) in a cluster gets its own unique cluster-wide IP address(one address per IP address family). This means that you don't need to explicitly create links between `Pods` and you almost never need to deal with mapping container ports to host ports.  
```

Comment:
Every [Pod](https://github.com/docs/concepts/workloads/pods/) in a cluster gets its own unique cluster-wide IP address(one address per IP address family).
This means that you don't need to explicitly create links between Pods and you
almost never need to deal with mapping container ports to host ports.

Following the same structure as was before will be more helpful.